src/IBM.WatsonDeveloperCloud/Http/Request.cs

```diff
@@ -109,6 +109,7 @@ public TaskAwaiter<IResponse> GetAwaiter()
         public async Task<T> As<T>()
         {
             HttpResponseMessage message = await this.AsMessage().ConfigureAwait(false);
+            var result = message.Content.ReadAsStringAsync().Result;

```

Comment:
@atilatosta I'm not sure I understand. Will we be able to get the json from the call above? To test I switched the call in the method in the abstraction to `AsString()` instead of `As<ClassifyTopLevelMultiple>()`. I am able to print the raw json within the method and deserialize into the object but the return object is still `ClassifyTopLevelMultiple`. Is there any way to get the raw json response as a string? 

```
private void ClassifyGet()
{
    var result = _visualRecognition.Classify(_imageUrl).AsString();
}
```

What do you think?